### PR TITLE
Remove dual-hosted info about kops and kubespray

### PR DESCRIPTION
**Pull request by tengqm**
Both tools have there own site and documentation. We can remove the details from the website, leaving only some pointers for users who really need to find out more.

related: #16091

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/_index.md [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
 weight: 30
+no_list: true
 ---
+
+There are many methods and tools for setting up your own production Kubernetes cluster.
+For example:
+
+- [kubeadm](/docs/setup/production-environment/tools/kubeadm/)
+
+- [kops](https://github.com/kubernetes/kops/): An automated cluster provisioning too.
```
Comment (windsonsea) [inline content/en/docs/setup/production-environment/tools/_index.md:12]:
```suggestion
- [kops](https://github.com/kubernetes/kops/): An automated cluster provisioning tool.
```

**Comment by sftim** [comment]
@tengqm this change might be helpful or unhelpful to readers. Would you be willing to open an issue where we can discuss the best approach?

**Comment by tengqm** [comment]
@sftim Related issue is https://github.com/kubernetes/website/issues/16091

**Comment by sftim** [comment]
OK, so what I'd like to do:
- link to https://kops.sigs.k8s.io/ for kops
- also track a frozen issue to see if the kops people are willing to put their docs inside https://k8s.io/docs/
  - if it's a no, it's a no

**Comment by tengqm** [comment]
Also related: #26910

File: content/en/docs/setup/production-environment/tools/_index.md [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+no_list: true
 ---
+
+There are many methods and tools for setting up your own production Kubernetes cluster.
+For example:
+
+- [kubeadm](/docs/setup/production-environment/tools/kubeadm/)
+
+- [kops](https://kops.sigs.k8s.io/): An automated cluster provisioning tool.
+  For tutorials, best practices, configuration options  and information on
```
Comment (kbhawkey) [inline content/en/docs/setup/production-environment/tools/_index.md:13]: nit: two spaces

**Comment by kbhawkey** [comment]
Hello @tengqm .
Have you reached out to the kOps and kubespray teams and let them know that their content is changing?
LGTM

**Comment by tengqm** [comment]
> Hello @tengqm . Have you reached out to the kOps and kubespray teams and let them know that their content is changing? LGTM

No. I didn't. But I believe dual-hosting is not a good thing for these contents.

**Comment by reylejano** [comment]
I'm good with this change. It is difficult to keep two sources up-to-date
/lgtm

**Pull request merged**